src/utils/ssr/getExperimentInfo.js

```diff
@@ -13,9 +12,14 @@ const getExperimentInfo = async (context, store) => {
     return;
   }
 
+  const user = await Auth.currentAuthenticatedUser();
+  const jwt = user.getSignInUserSession().getIdToken().getJwtToken();
+
+  console.log(`beautiful jwt ${jwt}`);
```

Comment:
I am sure it's beautiful, but we don't need to print it :)

Reply:
i wanted everyone to know, but i guess we can remove it if you insist :/